docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.8"
+
+services:
+  mysqldb:
+    platform: linux/arm64
+    image: mysql
+    restart: unless-stopped
+    volumes:
+      - ./mysqldata:/var/lib/mysql:rw,delegated
+    environment:
+      MYSQL_ROOT_PASSWORD: ${DATABASE_PASSWORD}
+      MYSQL_DATABASE: ${DATABASE_SCHEMA}
+    ports:
+      - "${DATABASE_LOCAL_PORT}:${DATABASE_DOCKER_PORT}"
+    healthcheck:
+      test: ["CMD-SHELL", "mysqladmin ping -h localhost -P ${DATABASE_DOCKER_PORT} -u ${DATABASE_USER} -p${DATABASE_PASSWORD}"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+
+  bookapp:
+    depends_on:
+      mysqldb:
+        condition: service_healthy
+    restart: on-failure
+    image: book-app
+    build: .
+    ports:
+      - "${SPRING_LOCAL_PORT}:${SPRING_DOCKER_PORT}"
+      - "${DEBUG_PORT}:${DEBUG_PORT}"
+    environment:
+      SPRING_APPLICATION_JSON: '{
+        "spring.datasource.url": "jdbc:mysql://mysqldb:${DATABASE_DOCKER_PORT}/${DATABASE_SCHEMA}?serverTimezone=UTC", 
+        "spring.datasource.username": "${DATABASE_USER}", 
+        "spring.datasource.password": "${DATABASE_PASSWORD}"
+      }'
```

Comment:
```suggestion
        "spring.datasource.url": "jdbc:mysql://mysqldb:${DATABASE_DOCKER_PORT}/${DATABASE_SCHEMA}?serverTimezone=UTC", 
        "spring.datasource.username": "${DATABASE_USER}", 
        "spring.datasource.password": "${DATABASE_PASSWORD}"
        "jwt.expiration": "${JWT_EXPIRATION}",
          "jwt.secret": "${JWT_SECRET}"
      }'
```
Does it work for you, was constantly getting an error creating jwtUtil?

Reply:
No, for me it works fine as I copy `.env` file in Docker image